contrib/azure/azuredeploy.json

```diff
@@ -7,7 +7,7 @@
 			"metadata": {
 				"description": "The base URI where artifacts required by this template are located. For example, if stored on a public GitHub repo, you'd use the following URI: https://raw.githubusercontent.com/deislabs/krustlet/master/."
 			},
-			"defaultValue": "https://raw.githubusercontent.com/deislabs/krustlet/master/"
+			"defaultValue": "https://raw.githubusercontent.com/deislabs/krustlet/master/contrib/azure"
```

Comment:
AHA! I see the problem now. It's missing a trailing slash in the URL.

Once you add that, the value should work. Re-testing.

Reply:
Ah my bad, I figured this was like `os.path.join`

Reply:
nah, ARM is a bit dumb at times :P